core/src/docs/upgrade.md

```diff
@@ -1,3 +1,24 @@
+# Upgrade to v0.47
+
+## Public API
```

Comment:
`BytesRange` is not part of public API.

Reply:
No. We expose it and I noticed that GreptimeDB uses this interface: https://github.com/GreptimeTeam/greptimedb/blob/2ade511f26066b2aa0ee138146965977210b40f2/src/object-store/src/layers/lru_cache/read_cache.rs#L248

Reply:
OK.